### PR TITLE
Add a script for adding GA tags to generated HTML

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
This can be run on a target directory before publishing to github

@jtattermusch

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by mehrdada** [comment]
Isn't it recommended to put GA script at the end of the file for loading speed purposes? (I'm not up to date with the current script snippet, but I remember that used to be the case)

**Comment by carl-mastrangelo** [comment]
@mehrdada Since it loads async, I don't think it matters much anymore.  The difference is practically immeasurable.

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by mehrdada** [comment]
Does this pass [`shellcheck`](https://www.shellcheck.net/)?

File: tools/doc/addga.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```
Comment (jtattermusch) [inline tools/doc/addga.sh]: let's rename to something more comprehensible.

`tools/api_reference/add_google_analytics.sh`?

- the `doc` sounds too general
- no one will know what `addga` is

Comment (carl-mastrangelo) [reply tools/doc/addga.sh]: Done.

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Pull request merged**